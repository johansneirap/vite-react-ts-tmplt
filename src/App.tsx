function App() {
  return <h1>Hello world template with Vite + React + TS</h1>;
}

export default App;
