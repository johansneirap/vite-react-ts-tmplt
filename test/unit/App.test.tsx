import '@testing-library/jest-dom';
import { render } from '@testing-library/react';
import App from 'App';

it('initial test', () => {
  const { container } = render(<App />);
  expect(container).toMatchSnapshot();
});
