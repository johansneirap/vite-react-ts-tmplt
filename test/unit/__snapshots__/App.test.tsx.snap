// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`initial test 1`] = `
<div>
  <h1>
    Hello world template with Vite + React + TS
  </h1>
</div>
`;
